Jenkinsfile

```diff
@@ -73,7 +70,7 @@ def runTests = { Map settings ->
     throw new Exception("Need Docker version to test, e.g.: `runTests(dockerVersion: '19.03.12')`")
   }
   if (!pythonVersion) {
-    throw new Exception("Need Python version being tested, e.g.: `runTests(pythonVersion: 'py2.7')`")
+    throw new Exception("Need Python version being tested, e.g.: `runTests(pythonVersion: 'py3.7')`")
```

Comment:
Here too `3.6`

Reply:
This one is the same as above. It's the version on which it will be tested. Not released.